### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by br1an-S**
I think it is a good time to merge what has been changed into master.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+*.pch
+
+# Compiled Dynamic libraries
+*.so
+*.dylib
+*.dll
+
+# Fortran module files
+*.mod
+*.smod
```
Comment (mccabe082) [inline .gitignore:24]: very thorough.  I would lose the fortran modules personally and I'm not sure what a few of the extensions correspond to but they do no harm what so ever.  Keep it like this if you want... you never know!

Comment (mccabe082) [reply .gitignore:24]: just realised this is a merge of dev to master. whoops silly me.  Obviously I've reviewed this b4 :P

**Review by mccabe082 — APPROVED** [review]
Go for it...

File: .gitignore (hunk 1 above)
Comment (br1an-S) [reply .gitignore:24]: Yeah ... Thats Cool.

**Review by John-Cozy — APPROVED** [review]
All these changes look good to me. We've already verified both the .bat and .sh files work. I'm assuming the CMake file's changes are suitable. The remaining changes are to the gitignore and readme.

**Pull request merged**